Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml

```diff
@@ -0,0 +1,35 @@
+- type: job
+  id: Paramedic
+  name: job-name-paramedic
+  description: job-description-paramedic
+  playTimeTracker: JobParamedic
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Medical
+      time: 1800
+  startingGear: ParamedicGear
+  icon: "Paramedic"
+  supervisors: job-supervisors-cmo
+  access:
+  - Medical
+  - Maintenance
+  - External
+  - Service
+  extendedAccess:
+  - Chemistry
+
+- type: startingGear
+  id: ParamedicGear
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitParamedic
+    back: ClothingBackpackMedicalFilled
+    shoes: ClothingShoesColorWhite
+    outerClothing: ClothingOuterCoatParamedicWB
+    head: ClothingHeadHatParamedicsoft
+    id: ParamedicPDA
+    ears: ClothingHeadsetMedical
+    pocket1: HandheldCrewMonitor
```

Comment:
Adding a crew monitor per paramedic is debatable but I don't mind it personally.